src/Facebook/FacebookClient.php

```diff
@@ -197,7 +197,7 @@ public function prepareRequestMessage(FacebookRequest $request)
      */
     public function sendRequest(FacebookRequest $request)
     {
-        if (get_class($request) === 'FacebookRequest') {
+        if (get_class($request) === 'Facebook\FacebookRequest') {
```

Comment:
It took me a while to remember why this `if` block is here. It's because the access token isn't required on the root request of a batch request. So this line might actually read better if it were changed to this:

```php
if (get_class($request) !== 'Facebook\FacebookBatchRequest') {
```

@yguedidi What are your thoughts?